test/runtime/Policies.go

```diff
@@ -1393,3 +1394,101 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(res.Output().String()).Should(ContainSubstring(allowedVerdict), "Policy trace did not contain %s", allowedVerdict)
 	})
 })
+
+var _ = Describe("RuntimeValidatedPolicyDropAllTests", func() {
```

Comment:
`RuntimeValidatedPolicies` creates a bunch of containers in the `BeforeAll` which are not needed for this test. Please read the container creation for this test carefully. `SampleContainersActions` creates containers with labels by default. Also please note that the container creation for` ingress` and `egress` is different.

Reply:
@manalibhutiyani the other containers can be in there without problems. You create a new container and you make the test based on that container. You should create a new `Context(`

Reply:
@eloycoto The problem is 
```
BeforeAll(func() {
		****
		areEndpointsReady := vm.WaitEndpointsReady()
		Expect(areEndpointsReady).Should(BeTrue(), "Endpoints are not ready after timeout")
	})
```
`vm.WaitEndpointsReady` will hang because in this case, we are always in `waiting-for-identity` state.


I have tried this out and this will hang for the DROP_ALL tests. Also, we are unnecessary creating containers which this test does not need.  I think we do not need to overkill the whole test-grouping process  and group tests logically. Please note that drop_all is not a policy based test.

Reply:
@eloycoto : This is what I get if I try to incorporate this as part of `RuntimeValidatedPolicies`

```
   /Users/manalibhutiyani/go/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:312

    Unable to delete all policies
    Expected
        <bool>: false
    to be true

    /Users/manalibhutiyani/go/src/github.com/cilium/cilium/test/runtime/Policies.go:441
------------------------------
RuntimeValidatedPolicies DROP_ALL Egress Policy test 
  DROP_ALL Egress Policy test

```

this is being hit in 
```
AfterEach(func() {
		vm.PolicyDelAll().ExpectSuccess("Unable to delete all policies")
	})
```
Since DROP_ALL test do not have any policies, this global `AfterEach` simply fails.